### PR TITLE
EOS VISION x REDDIT

### PR DESCRIPTION
**Pull request by Creamyvanill**
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
> Please delete these lines that start with a `>`

### For which grant [level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels) are you applying?

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 4 approvals

### Application Checklist

- [x] The [application template]([xxxx](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/application-template.md)) has been copied, renamed (`my_project_name.md`) and updated.
- [x] The application file (`my_project_name.md`) has no spaces in its name, ends with the `.md` extension has been placed in the `applications` directory.
- [x] I have read and understood the:
  - [x] [FAQs](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/faq.md), 
  - [x] [application guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/grant_guidelines_per_category.md) and 
  - [x] [announcement guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/announcement-guidelines.md).
- [x] An EOS address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] This project has NOT been part of a token sale or NFT sale.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/eosnetworkfoundation/grant-milestones#process).

### How Did You Hear About our grants program?

- [x] Social Media
- [ ] EOSNetwork.com
- [ ] Personal Recommendation
- [ ] ENF Fireside Chat
- [ ] Investor/VC
- [ ] Online Search
- [ ] Hackathon
- [ ] Other: _______

### CODE REVIEW

**Comment by nsjames** [comment]
Hi @Creamyvanill 

I've created a PR into your repository with some minor changes for structure. Please review and merge it in so it will reflect here. Thanks!

**Comment by Creamyvanill** [comment]
Hi! Mr @nsjames 

Thank you very much for the corrections, I have already been able to merge it, have a nice night!

**Review by nsjames — APPROVED** [review]

File: applications/eosreddit.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  - This is a place for you to manage expectations and to clarify any limitations that might not be obvious
+  - 3 PILLARS VISIONS REDDIT.
+1.Make different daily Posts regarding the EOS Ecosystem: Maintaining activity within Reddit is one of the most important ways to eradicate FUD, demonstrating that it is an active community and that good opportunities, good projects are being created and there are new news for read, share and comment at all times.
+
+2. Increase in daily moderation: A moderation of all the SPAM that contains the official EOS reddit will be carried out, as well as the active moderation queue that may exist, it is worth mentioning that the SPAM queue has accumulated in recent months due to the number of bots that enter our Reddit.
+
+3. Reddit-Discord Reward System: Based on the “Tip.cc” Reward system in Discord, and the EOS BEES idea of ​​performing “SWARM PAID” to different important Twitter Posts, we will bring this idea to Reddit, dedicate 2 minutes a day to Vote on our Reddit, check out the “VISIONS REDDIT” channel where we will upload active posts daily, here you can vote and post in our thread that you have voted so that you can earn daily rewards.
```
Comment (DenisCarriere) [inline applications/eosreddit.md:49]: @Creamyvanill this `3.` section is out of scope of the current milestones, when first reading it seems like it was included in the initial grant request.

Perhaps could state that this section is in (Future Plan).

**Review by DenisCarriere — APPROVED** [review]
Do strongly believe that EOS Vision can fulfill the scope of the propose Milestone 1.

File: applications/eosreddit.md (hunk 1 above)
Comment (Creamyvanill) [reply applications/eosreddit.md:49]: Good point, I will update it, for a future plan, just as I will update that the ENF already has its account on Reddit. @DenisCarriere

**Pull request merged**